src/MgvLib.sol

```diff
@@ -187,25 +188,3 @@ interface IMgvMonitor {
 
   function read(address outbound_tkn, address inbound_tkn) external view returns (uint gasprice, uint density);
 }
-
-interface IERC20 {
```

Comment:
So this is changing audited code, but it is just moving stuff around. Is that ok?

Reply:
Damn I misremembered that MgvLib was not audited. Yes I guess :/